### DISCUSSION TITLE
Mapping: Song file format

### DISCUSSION THREAD

**Discussion by AhegaHOE** [ppy/osu#22393] [Ideas]
In the current build of the game (`2023.123.0-lazer`) there is no indication on what file formats can be used when creating a new beat map.

Here are the steps to reproduce it:

1. Go into Edit-Mode
2. Select in the Audio-Track section a folder where there is for example only a flac file inside.

I think there should be some kind of hint what file formats are supported.

**Comment by integerrr** [comment]
According to [the wiki](https://osu.ppy.sh/wiki/en/Beatmap):

> the song itself, stored in MP3 or Ogg format.


It's also highly unlikely a flac file will comply with criterion regarding the data rate for the audio file. 

At the moment what would happen if you were to select the flac file?

**Reply by AhegaHOE** [reply]
You can not select a flac file. They simple don't show up.

Only MP3 and OGG show up and there is no indication in the file browser when you want to select a song.

![osu_2023-01-24_14-10-27](https://user-images.githubusercontent.com/73001005/214302195-1482f447-3705-4f3d-bed7-ffb0341ec4a3.jpg)

It would be nice if somewhere is noted which files are shown.

Maybe something like. `*.MP3, *.OGG`
